network/policy/policy_windows_test.go

```diff
@@ -57,4 +143,80 @@ var _ = Describe("Windows Policies", func() {
 			Expect(string(generatedPolicy.Settings)).To(Equal(expectedPolicy))
 		})
 	})
+
+	Describe("Test AddNATPolicyV1", func() {
+		It("Should marshall the NAT policy v1 correctly", func() {
+			expectedPolicy := `{"Type":"OutBoundNAT","Destinations":["168.63.129.16"]}`
+
+			generatedPolicy, err := AddDnsNATPolicyV1()
+			Expect(err).To(BeNil())
+			Expect(string(generatedPolicy)).To(Equal(expectedPolicy))
+		})
+	})
+
+	Describe("Test AddNATPolicyV2", func() {
+		It("Should marshall the NAT policy v2 correctly", func() {
+			vip := "vip"
+			destinations := []string{"192.168.1.1", "192.169.1.1"}
+
+			expectedPolicy := `{"VirtualIP":"vip","Destinations":["192.168.1.1","192.169.1.1"]}`
+
+			generatedPolicy, err := AddNATPolicyV2(vip, destinations)
+			Expect(err).To(BeNil())
+			Expect(string(generatedPolicy.Settings)).To(Equal(expectedPolicy))
+		})
+	})
+
+	Describe("Test GetHcnEndpointPolicies", func() {
+		It("Should marshall the policy correctly", func() {
+			testPolicies := []Policy{}
+
+			rawPortMappingPolicy, _ := json.Marshal(&hcn.PortMappingPolicySetting{
+				ExternalPort: 8008,
+				InternalPort: 8080,
+			})
+
+			portMappingPolicy, _ := json.Marshal(&hcn.EndpointPolicy{
+				Type:     hcn.PortMapping,
+				Settings: rawPortMappingPolicy,
+			})
+
+			hnsPolicy := Policy{
+				Type: PortMappingPolicy,
+				Data: portMappingPolicy,
+			}
+
+			testPolicies = append(testPolicies, hnsPolicy)
```

Comment:
I know we've tested most of the policies from the witch statement in `GetHcnEndpointPolicies` individually, but can we have a test where we pass in more than 1 endpoint policy to confirm they are being looped over and added properly?